### PR TITLE
Thang/fix mlflow error

### PR DESCRIPTION
**Pull request by HDThang**
# What does this PR do?
I know there are a lot of files but I just replace the same line of code in them.

-Fix the error raise by mlflow for duplicate experiment ids when run the same model again.
-Fix some minor bugs

### CODE REVIEW

File: examples/pytorch/audio-classification/run_audio_classification.py [hunk 1]
```diff
@@ -382,18 +385,32 @@ def compute_metrics(eval_pred):
         tokenizer=feature_extractor,
     )
 
+    # mlflow initial
+    experiment_name ='translateion-{}'.format(model_args.model_name_or_path)
```
Comment (yeomni) [inline examples/pytorch/audio-classification/run_audio_classification.py:389]: typo

File: examples/pytorch/audio-classification/run_audio_classification.py [hunk 2]
```diff
@@ -382,18 +385,32 @@ def compute_metrics(eval_pred):
...
             checkpoint = training_args.resume_from_checkpoint
         elif last_checkpoint is not None:
             checkpoint = last_checkpoint
-        train_result = trainer.train(resume_from_checkpoint=checkpoint)
-        trainer.save_model()
-        trainer.log_metrics("train", train_result.metrics)
-        trainer.save_metrics("train", train_result.metrics)
-        trainer.save_state()
+        
+        with mlflow_runner:
```
Comment (yeomni) [inline examples/pytorch/audio-classification/run_audio_classification.py:402]: Looks like it only does one time log_metric. Please make sure it should record loss/throughput every epoch at least.

File: examples/pytorch/language-modeling/.nfs00000000864964a600000492 [hunk 3]
```diff
@@ -0,0 +1,3 @@
+gpt2 8
```
Comment (yeomni) [inline examples/pytorch/language-modeling/.nfs00000000864964a600000492:1]: This filename should be renamed or deleted if it was created by incident.

File: examples/pytorch/token-classification/.nfs0000000087dad97000000491 [hunk 4]
```diff
@@ -0,0 +1,3 @@
+bert-base-uncased 64
```
Comment (yeomni) [inline examples/pytorch/token-classification/.nfs0000000087dad97000000491:1]: This one as well.

File: examples/pytorch/text-classification/README.md [hunk 5]
```diff
@@ -191,7 +191,7 @@ python run_xnli.py \
   --per_device_train_batch_size 32 \
   --learning_rate 5e-5 \
   --num_train_epochs 2.0 \
-  --max_seq_length 128 \
+  -x_seq_length 128 \
```
Comment (yeomni) [inline examples/pytorch/text-classification/README.md:194]: Was this a mistake?

File: examples/pytorch/audio-classification/run_audio_classification.py (hunk 1 above)
Comment (hongjoomoreh) [reply examples/pytorch/audio-classification/run_audio_classification.py:389]: + f-string is usually better thatn 'format' method.

File: examples/pytorch/audio-classification/run_audio_classification.py [hunk 6]
```diff
@@ -41,6 +41,9 @@
 from transformers.utils import check_min_version, send_example_telemetry
 from transformers.utils.versions import require_version
 
+# Initialize MLFlow
+import mlflow
+mlflow.set_tracking_uri("http://127.0.0.1:5000")
```
Comment (hongjoomoreh) [inline examples/pytorch/audio-classification/run_audio_classification.py:46]: Shouldn't `tracking_uri` be a parameter with default value?
Or can we convince that this script will always use the same uri regardless of future change?

For example, in the future we can use external cloud for mlflow, or our remote server (other than script-running-server). If URI is already configurable or still fine for future changes, then please ignore this comment

Comment (dohyung-kim-moreh) [reply examples/pytorch/audio-classification/run_audio_classification.py:46]: Model Runner sets up 'MLFLOW_TRACKING_URI'.
There's no need to call set_tracking_uri in a training/inference script.

File: examples/pytorch/audio-classification/run_audio_classification.py [hunk 7]
```diff
@@ -382,18 +385,32 @@ def compute_metrics(eval_pred):
         tokenizer=feature_extractor,
     )
 
+    # mlflow initial
+    experiment_name ='translateion-{}'.format(model_args.model_name_or_path)
+    current_experiment = mlflow.set_experiment(experiment_name)
+    experiment_id=current_experiment.experiment_id
+    mlflow_runner = mlflow.start_run(run_name=model_args.model_name_or_path, experiment_id=experiment_id)
```
Comment (dohyung-kim-moreh) [inline examples/pytorch/audio-classification/run_audio_classification.py:392]: Model Runner sets up all the run_id and run_name, experiment arguments.
Just call mlflow.start_run().

**Review by thuc-moreh — APPROVED** [review]

**Comment by yeomni** [comment]
Why is this PR still on hold? Looks like it should have been merged 2 months ago. @HDThang

**Comment by HDThang** [comment]
Close due to duplicate with another branch
